lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -30,6 +30,28 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
         resources "/users", MyAppWeb.UserController, only: [:index, :create, :show]
         get "/openapi", OpenApiSpex.Plug.RenderSpec, :show
       end
+
+  ## Runtime Configuration
+
+  SwaggerUI also accepts functions as parameters, allowing runtime configurations.
+  Because of limitations of `router.ex` file, the functions can be defined only as: `{module, function_name}`.
```

Comment:
The SwaggerUI module is a Plug, it can be used even by a non-phoenix app specifying its routes in a `router.ex`.

Reply:
Ah my bad, I will update the documentation.